code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -176,21 +205,30 @@
 		return
 
 	var/staggerslow_stacks = 2
+	var/paralyzesecs = 1
```

Comment:
Isn't this life ticks?

Reply:
the value should be same as how sneak attack was before this but modularized now.